test-app/app/routes/application.ts

```diff
@@ -0,0 +1,62 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
```

Comment:
The service should infer types from the source

```suggestion
import Route from '@ember/routing/route';
import { inject as service } from '@ember/service';
import { type LiveAgentService } from 'ember-liveagent';
// or
// import type LiveAgentService from 'ember-liveagent/services/liveagent';
```